### PR TITLE
Swipe menu ng build issue

### PR DESCRIPTION
**Pull request by SeanKelly369**
Now builds and runs successfully

### CODE REVIEW

**Review by SeanKelly369 — COMMENTED** [review]
merging to master

File: src/app/app.module.ts
Comment (darklight365) [inline src/app/app.module.ts:1]: @SeanKelly369 + @farfromrefug  - would this commit not break every other angular plugin in the nativescript-community that makes use of this plugin seed?

For example take a look at the [install.module](https://github.com/nativescript-community/ui-material-components/blob/a8a588bfb9658b3974901223c74e8a1442af20d0/demo-snippets/ng/install.module.ts#L29) in this plugin (which also relies on this angular seed project):
https://github.com/nativescript-community/ui-material-components

Comment (farfromrefug) [reply src/app/app.module.ts:1]: @darklight365. Yes it Will be used in  a repos now. 
But why would it break ?

Comment (darklight365) [reply src/app/app.module.ts:1]: @farfromrefug - The module now hard code includes the ui-collectionview components and does no longer dynamically include the COMPONENTS via the plugins install.module.

Check here: https://github.com/nativescript-community/plugin-seed-demo-ng/blob/82586511e9f8cdb46811e4a748a2ddf35a8503cb/src/app/app.module.ts#L18

Comment (farfromrefug) [reply src/app/app.module.ts:1]: Good catch it is bad !

Comment (darklight365) [reply src/app/app.module.ts:1]: Prior to this commit the seed project would import the COMPONENTS from the demo-snippets install.module, as seen here:

https://github.com/nativescript-community/ui-material-components/blob/a8a588bfb9658b3974901223c74e8a1442af20d0/demo-snippets/ng/install.module.ts#L29

With this change: the only components included in the angular test project is the ui-collectionview components.

Comment (farfromrefug) [reply src/app/app.module.ts:1]: That was a try to fix build errors on angular demos. 
Buy you got it to work in swipemenu without this? I mean did you revert this to have it work in swipemenu?

Comment (darklight365) [reply src/app/app.module.ts:1]: No, I did a clean checkout from master (force updated my branch). So the SwipeMenu changes worked with the current ng project seed.

I would need to perform some additional tests to see if reverting the ng seed app.module has any build issues still.

Comment (darklight365) [reply src/app/app.module.ts:1]: @farfromrefug - I've made the Angular seed generic again with this PR: https://github.com/nativescript-community/plugin-seed-demo-ng/pull/6

The other piece of the puzzle is to correctly export the SwipeMenuComponent in the demo-snippets/ng, PR for that is:
https://github.com/nativescript-community/ui-collectionview/pull/80

cc: @SeanKelly369 for info

**Pull request merged**